app/front/scripts/controllers/provide-metadata.js

```diff
@@ -25,36 +32,41 @@ angular.module('Application')
           end: fiscalPeriod ? fiscalPeriod.end : ''
         };
 
-        $scope.$watch('attributes.title', function() {
```

Comment:
But in that watch we check `if ((newValue === oldValue))`, which basically means that if the object is the same one (regardless of property changes) then we won't do anything, no?